vta/include/vta/hw_spec.h

```diff
@@ -365,6 +365,9 @@ extern "C" {
 #define VTA_UOP_ALU_1_0 (VTA_UOP_ALU_0_1 + 1)
 /*! GEMM Micro-op end position of the inp_idx field */
 #define VTA_UOP_ALU_1_1 (VTA_UOP_ALU_1_0 + VTA_LOG_INP_BUFF_DEPTH - 1)
+/*! PAGE SIZE */
```

Comment:
I'm not fully convinced that this should be in this file. Instead I'd recommend moving these macro definitions in `vta/include/vta/driver.h`

Reply:
fixed.